### ISSUE TITLE
Show quests on higher zooms

### ISSUE THREAD

**Issue by BANOnotIT** [wtimme/OSM-Completionist#66]
# User Story

> As a bus mapper, I want see all quests near me on high speed so I can react and check something without leaving the bus.

# Acceptance Criterias

I don't know right now. In streetcomplete that worked and here such quests are
1. less attraction
2. couldn't be reacted quickly

# Alternatives Considered

It'll be cool I think to have separate mode for quests only. But it can by separate app as well...

# Unhappy Cases

Some users using this app as a normal editor could be unhappy with cluttererd hight zooms.

# Technical Steps (optional)

It'll be good, to tap on some quest, app autozooms to it and after answer zoom back.

**Comment by bencook** [comment]
As a new user, I found it very difficult to discover quests in my area.  It was not obvious until looking very closely that there were any available quests.  Streetcomplete makes it much easier to see nearby quests at a higher zoom level.

**Comment by wtimme** [comment]
Thanks for the input! I absolutely agree. 

At the moment, the annotations are tightly coupled with the display logic for notes/ways/relations.

I tried to come up with a more generic approach to displaying annotations on the map (cp. #52), but that turns out to be quite some work.

I decided to create a new app that makes use of [Tangram-es](https://github.com/tangrams/tangram-es), which allows for more customization of the look-and-feel. I tried to reuse a lot of the things from [StreetComplete](https://github.com/westnordost/StreetComplete/), also matching its visual appearing. Take a look:

![osm-surveyor](https://user-images.githubusercontent.com/1681085/83308774-d594f000-a207-11ea-9b84-71e49d344bc8.png)

That being said, I'm happy for any pull requests for OSM-Completionist (or, even better, the mother ship - GoMap) that introduce this feature.

**Comment by bryceco** [comment]
I agree that for a StreetComplete type app this is a better approach, overlaying quest markers on a layer that’s independent of the OSM data, and using overpass to gather and display the quests. I think there’s a lot of potential here either as a standalone app or as a Go Map!! layer.

**Comment by pqhf5kd** [comment]
> I decided to create a new app that makes use of [Tangram-es](https://github.com/tangrams/tangram-es), which allows for more customization of the look-and-feel. I tried to reuse a lot of the things from [StreetComplete](https://github.com/westnordost/StreetComplete/), also matching its visual appearing. Take a look:

Just to confirm @wtimme, you're shelving OSM-Completionist and working on OSM-Surveyor instead?

**Comment by wtimme** [comment]
> Just to confirm @wtimme, you're shelving OSM-Completionist and working on OSM-Surveyor instead?

Thanks for asking, @pqhf5kd! I've just added a deprecation notice to the README and do not intend to continue working on OSM-Completionist.

The main reason is the lack of contributors and the conflicts that I run into every time I merge `master` from the original GoMap!! repository. When I started out with the project, I assumed that these merges would be more or less easy peasy, since in my (professional/job) past, I have had good experience with these kind of forks. But, sadly, neither my original MVP approach (cp. bryceco/GoMap#221) nor this fork proved to be viable.

I have therefore started with OSM-Surveyor, which at this point is a proof-of-concept for me for translating the StreetComplete logic from Java/Kotlin to Swift, in order to reuse the thinking that they did and not to reinvent the wheel.

As for the original request in this GitHub issue, as pointed out, I think this needs to be implemented directly in GoMap instead of OSM-Completionist, since the former is doing the heavy lifting with the rendering.
